### PR TITLE
doc: add maintainers doc for releasing

### PR DESCRIPTION
**Pull request by tormath1**
Everything is in the title.

### CODE REVIEW

**Review by t-lo — APPROVED** [review]
Looks good, thank you Mathieu!

File: README.md [hunk 1]
```diff
@@ -177,3 +177,13 @@ In case you have an existing Torcx image you can convert it with the `convert_to
...
+
+## For maintainers: how to release?
+
+CI can be kicked-off by overriding the `latest` tag. The `latest` release artifacts will be updated consequently here: https://github.com/flatcar/sysext-bakery/releases/tag/latest
+```
+git rebase origin/main
+git tag -d latest
+git tag -as latest
+git push origin --force latest
+```
```
Comment (pothos) [inline README.md:189]: Or maybe we can change the CI to re-release on every `main` change

Comment (t-lo) [reply README.md:189]: Would make sense to me. I'm working on a build automation feature to build releases based on a list of versions in a text file, and include all sysexts from the previous "latest" release. Your proposal would be a great follow-up task to that PR.

Comment (t-lo) [reply README.md:189]: ==> https://github.com/flatcar/sysext-bakery/pull/40

Comment (t-lo) [reply README.md:189]: Also #42 .

**Pull request merged**